packages/fxa-content-server/app/scripts/templates/partial/brand-messaging.mustache

```diff
@@ -0,0 +1,37 @@
+{{#showBanner}}
+<div id="banner-brand-message" class="absolute w-full top-0 left-0">
+    <div class="flex justify-center p-2 brand-banner-bg">
+        {{#showPrelaunch}}
+        <div class="flex"/>
+            <div class="flex-none m-logo relative mt-1 mr-4 mb-2 bg-black">
+                <div>
+                    <img src="../../../images/brand_messaging/icon_m.png" alt="mozilla m logo" />
```

Comment:
I think you will need to have the image in background css like [this](https://github.com/mozilla/fxa/blob/8cea71c67a7eef381751be46c1a52fcf33221a90/packages/fxa-content-server/app/styles/modules/_custom-rows.scss#L182). Otherwise this would work locally but not stage/prod.

Reply:
I've switched this to use base64 encoded image data strings. Seems cleaner in general. Ive also done this the react component as well.